client/dashboard/sites/overview-latest-activity-card/index.tsx

```diff
@@ -46,8 +46,8 @@ const fields: Field< ActivityLogEntry >[] = [
 
 function getActivityLogUrl( site: Site ) {
 	return isSelfHostedJetpackConnected( site )
-		? `https://cloud.jetpack.com/activity-log/${ site.slug }`
-		: `/activity-log/${ site.slug }`;
+		? `https://cloud.jetpack.com/activity-log/size=${ site.slug }`
```

Comment:
Is this change correct? The URL looks weird...

Reply:
Fixed